.lycheeignore

```diff
@@ -0,0 +1,2 @@
+https://metamask.zendesk.com/hc/en-us/articles/360015489331-How-to-import-an-account#:~:text=Click%20the%20circle%20icon%20at
+https://metamask.zendesk.com/hc/en-us/articles/360015488891-How-to-reset-an-account
```

Comment:
I wonder what this file is used for.

Reply:
@AtelyPham it acts as ignore file for the link checker. Since these links are flagged as forbidden in the CI environment the link check will fail yet these links are not broken and accessible.

Reply:
See readme here: https://github.com/lycheeverse/lychee-action#excluding-links-from-getting-checked